nixos/modules/services/web-apps/dex.nix

```diff
@@ -70,7 +87,8 @@ in
     systemd.services.dex = {
       description = "dex identity provider";
       wantedBy = [ "multi-user.target" ];
-      after = [ "networking.target" ] ++ (optional (cfg.settings.storage.type == "postgres") "postgresql.service");
+      after = [ "networking.target" ]
+        ++ (optional (cfg.mount_postgresql) "postgresql.service");
```

Comment:
```suggestion
        ++ optional (cfg.mount_postgresql) "postgresql.service";
```

Reply:
Always a bit nervous with precedence working against me. But this change seems OK...